docs/docs/stylesheets/extra.css

```diff
@@ -0,0 +1,14 @@
+:root > * {
+    --md-primary-fg-color: #a599ff;
+    --md-accent-fg-color: #B0A5FF;
```

Comment:
the first one is the "main KPOps color", right? What's the second one? Is it a slightly darker shade? consider adding a comment here for explanation